zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 0.1.91
+
+### Breaking changes
+
+- Firmware updates may no longer work properly in Home Assistant versions 2023.9.x and earlier due to a breaking change upstream that couldn't be made backwards compatible. They will be fixed in 2023.10.0 .
+
+### Features
+
+- Detect an unresponsive stick and reset it
+- The default time after which battery-powered devices with no pending commands are sent back to sleep is now 250 ms (down from 1000ms)
+- We no long rebuild the @serialport/bindings-cpp package from source
+
+### Bug fixes
+
+- A bug in the 7.19.x SDK has surfaced where the controller gets stuck in the middle of a transmission. Previously this would go unnoticed because the failed commands would cause the nodes to be marked dead until the controller finally recovered. Since v11.12.0 however, Z-Wave JS would consider the controller jammed and retry the last command indefinitely. This situation is now detected and Z-Wave JS attempts to recover by soft-resetting the controller when this happens.
+- Removed auto-disabling of soft-reset capability
+- Default to RF protection state Unprotected if not given for Protection CC V2+
+

```

Comment:
```suggestion
- Devices that send notifications from endpoints, e.g. Aeotec Wallmote, are now properly supported
```
(at least on the driver side, what about HA?)

Reply:
this will be supported in 2023.10 although thanks for reminding me, I missed that